docs/user/03-traces.md

```diff
@@ -522,4 +522,4 @@ If you just want to see traces for one particular request, you can manually forc
 
 **Cause**: Gateway cannot receive spans at the given rate.
 
-**Remedy**: Manually scale out the gateway by increasing the number of replicas for the Trace collector. See [Module Configuration and Status](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
+**Remedy**: Manually scale out the gateway by increasing the number of replicas for the Trace gateway. See [Module Configuration and Status](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
```

Comment:
```suggestion
**Remedy**: Manually scale out the gateway by increasing the number of replicas for the trace gateway. See [Module Configuration and Status](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
```

For consistency. It's lower case everywhere else.

Reply:
- line 49 stays unchanged, there we talk about the Otel Collector that gets deployed
- Standardized all occurrences of "Otel collector" to use capitalized "C"